src/main/resources/application.yaml

```diff
@@ -84,7 +84,7 @@ idam:
     url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
     pbaFromUserAccountMap: false
   s2s-authorised:
-    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac,aac_manage_case_assignment,divorce_frontend}
+    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac,aac_manage_case_assignment,divorce_frontend,am_org_role_mapping_service}
   api.url: ${idam_url:https://idam-api.aat.platform.hmcts.net}
 
```

Comment:
am_org_role_mapping_service - is not live now so it cant be added to application yaml.. we need to add this to flux config and enable this in lower env only
@AdilOoze  - can u add this to flux config for aat and demo?

Reply:
@shashank-rastogi sure, I will add `am_org_role_mapping_service` to flux config for `aat` and `demo`

Reply:
@shashank-rastogi here is the Pull Request for the above: https://github.com/hmcts/cnp-flux-config/pull/7601